lib/index.js

```diff
@@ -51,9 +51,8 @@ module.exports = function(options) {
       $('code').each(function() {
 
         var $this = $(this);
-        var className = $this.attr('class');
-
-        var targets = className ? className.split('language-') : ['', 'markup'];
+        var className = $this.attr('class') || '';
+        var targets = className.split('language-');
```

Comment:
Felt having a fallback value for `className` was easier too read than an empty array for `target` as proposed earlier.